javascript_nodejs/14.nlp-with-dispatch/README.MD

```diff
@@ -0,0 +1,127 @@
+    ```bash
+    cd javascript_nodejs/14.nlp-with-dispatch
+    ```
+- Point to the MyGet feed 
+    ```bash
+    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    ```
+- Install modules 
+    ```bash
+    npm i 
+    ```
+- Install required tools - to successfully setup and configure all services this bot depend on, you need to install the MSBOT, LUIS, QnAMaker, Ludown, Dispatch CLI tools. 
+    ```bash
+    npm i -g msbot luis-apis qnamaker ludown botdispatch
+    ```
+- Configure required LUIS, QnA Maker and Dispatch services. See [here](#configure-services)
+- Once services are configured, run the sample 
+    ```bash
+    npm start
+    ```
+- To reset registry, you can do
+    ```bash
+    npm config set registry https://registry.npmjs.org/
+    ```
+
+# Configure services
+This sample relies on [LUIS](http://luis.ai), [QnA Maker](http://qnamaker.ai) and [Dispatch](http://github.com/microsoft/botbuilder-tools//tree/master/packages/Dispatch) services. 
+
+## Configure the LUIS service
+To create required LUIS applications for this sample bot, 
+- Create an account with [LUIS](https://luis.ai). If you already have an account, login to your account.
+- Click on your name on top right corner of the screen -> settings and grab your authroing key.
+
+To create the LUIS application this bot needs and update the .bot file configuration, in a terminal, 
+- Clone this repository
+- Navigate to javascript_nodejs\14.nlp-with-dispatch
+- Run the following command
+```bash 
+> ludown parse toluis --in dialogs/homeAutomation/resources/homeautomation.lu -o cognitiveModels -n homeautomation.luis
+
+> ludown parse toluis --in dialogs/weather/resources/weather.lu -o cognitiveModels -n weather.luis
+
+> luis import application --in cognitiveModels\homeautomation.luis --authoringKey <YOUR-LUIS-AUTHORING-KEY> --endpointBasePath https://westus.api.cognitive.microsoft.com/luis/api/v2.0 --msbot | msbot connect luis --stdin --name homeautomation.luis
+
+> luis import application --in cognitiveModels\weather.luis --authoringKey <YOUR-LUIS-AUTHORING-KEY> --endpointBasePath https://westus.api.cognitive.microsoft.com/luis/api/v2.0 --msbot | msbot connect LUIS --stdin --name weather.luis
+```
+
+Note: Should you so desire, you can create the LUIS application in a region of your choice. See [here](https://docs.microsoft.com/en-us/azure/cognitive-services/LUIS/luis-reference-regions) to learn more about available LUIS authoring regions. You can specify a different region (westus or westeurope or australiaeast) via https://**LUIS-Authoring-Region**.api.cognitive.microsoft.com/luis/api/v2.0 in the above command.
+
+## Train and publish the LUIS models 
+You need to train and publish the LUIS models that were created for this sample to work. You can do so using the following CLI commands
+
+```bash
+> msbot get service --name homeautomation.lUIS | luis train version --wait --stdin
```

Comment:
I'm down with "LUIS" or "luis". But the reverse case is odd ("lUIS")

Reply:
Fixed.